tests/jeos/glibc_locale.pm

```diff
@@ -97,6 +97,7 @@ sub test_users_locale {
     ($ldd_help_string_expected eq $ldd_help_string_updated) or die "Unexpected locale settings, glibc test strings are not the same!\n";
     enter_cmd("exit");
 
+    reset_consoles;
```

Comment:
As we talked, the module works fine for sle15+ and opensuse. I am still in the doubt of breaking this as it is also used with community JeOS. So let's remove this to be safe and we can finally merge this pr